kb/temporal-platform-limits-sheet.md

```diff
@@ -32,9 +31,10 @@ Unless otherwise stated, Event limits apply to all Events occurring simultaneous
   - We warn at 10,000 Events: `history size exceeds warn limit.`
   - We error at 50,000 Events: [`history size exceeds error limit.`](https://github.com/temporalio/temporal/blob/v1.7.0/service/history/workflowExecutionContext.go#L1204)
   - This is configurable with [`HistoryCountLimitError` and `HistoryCountLimitWarn`](https://github.com/temporalio/temporal/blob/v1.7.0/service/history/configs/config.go#L382-L383), if you know what you are doing.
-  - We fail the following Commands at 50,000 Events for OSS and 2,000 Events for Cloud:
-    - `ScheduleActivityTask`
-    - `SignalExternalWorkflowExecution`
-    - `RequestCancelExternalWorkflowExecution`
-    - `StartChildWorkflowExecution`
+- We fail the following Commands at 50,000 Events for OSS and 2,000 Events for Cloud:
+  - `ScheduleActivityTask`
+  - `SignalExternalWorkflowExecution`
+  - `RequestCancelExternalWorkflowExecution`
+  - `StartChildWorkflowExecution`
```

Comment:
I think what @yiminc is saying is that we fail the following Commands at 2,000 **concurrent** Actions (note this NOT the same as Events) ?

Reply:
okay I gotcha